spec/graphql/mutations/stories/delete_story_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+module Mutations
+  module Stories
+    RSpec.describe DeleteStory, type: :request do
+      def query
+        <<~GQL
+          mutation {
+          deleteStory(
+            input: {
+              id: "#{@story.id}",
+            }
+          ) { story {
+              id
+              title
+            }
+          }
+        }
+        GQL
+      end
+
+      def bad_query
+        <<~GQL
+          mutation {
+          deleteStory(
+            input: {
+              id: "10900",
+            }
+          ) { story {
+              id
+              title
+            }
+          }
+        }
+        GQL
+      end
+
+      it 'deletes a story' do
+        stories = create_list(:story, 5)
+        @story = create(:story, title: 'Deleted')
```

Comment:
Not sure why we have a global @story variable here. It also doesn't seem to be used again in this test. Does it need to be an @?

Reply:
Removed, nice catch.